### ISSUE TITLE
Use different element-name instead of "Other" in XML-Output

### ISSUE THREAD

**Issue by shamrockDE** [Massi-X/freepbx-phonemiddleware#5]
First thanks a million for your project. I was looking for a way to implement our existing family-phonebook from Nextcloud via carddav in FreePBX and the Snom-Phones and your solution works.

**Is your feature request related to a problem? Please describe.**
As described in issue #3 i am using a Dect-Station m300 from Snom. The xml-phonebook for Snom accepts 3 elements: "Telephone", "Mobile" and "Office". The output of this projects produces "Telephone", "Mobile" and "Other".

**Describe the solution you'd like**
It would be great, if the element name "Other" in the XML-Output could be defined in the User-interface, to change it to "Office".

**Describe alternatives you've considered**
I changed it in the code directly, and it works ;-)

The output must look like this:
```
<?xml version="1.0" encoding="UTF-8"?>
<IPPhoneDirectory>
    <DirectoryEntry>
        <Name>John Doe</Name>
        <Telephone>123</Telephone>
        <Office>345</Office>
        <Mobile>123</Mobile>
    </DirectoryEntry>
    <DirectoryEntry>
        <Name>Ray Albert</Name>
        <Telephone>8971</Telephone>
        <Office>1233</Office>
        <Mobile>1233</Mobile>
    </DirectoryEntry>
</IPPhoneDirectory>
```

**Comment by Massi-X** [comment]
Hi and thank you again! Will add it in the next version because right now I'm working on #4!

**Comment by Massi-X** [comment]
Fixed in v1.0.6
